Sources/LanguageServerProtocolTransport/JSONRPCConnection.swift

```diff
@@ -551,21 +435,22 @@ public final class JSONRPCConnection: Connection {
   /// - Important: Must be called on `queue`
   private func send(data dispatchData: DispatchData) {
```

Comment:
Should we use `Data` here instead of `DispatchData` now that we’re no longer using `DispatchIO` for stdin/stdout writing. Would do it in a follow-up PR to keep this one as minimal as possible though.